src/main/java/frc/robot/subsystems/scoring/ScoringSubsystem.java

```diff
@@ -284,7 +290,10 @@ public void periodic() {
                                 FieldLocations.RED_STAGE))) {
             aimerIo.setAngleClampsRad(0, 0);
         } else {
-            aimerIo.setAngleClampsRad(0, ScoringConstants.aimMaxAngleRadians);
+            double elevatorLimit =
+                    (Math.PI / 2 / 0.45) * (elevatorPositionSupplier.get() - 0.45) + Math.PI / 2;
```

Comment:
What does this equation do / can it be documented better to explain what each of the magic numbers mean (pi / 2, 0.45)